cmd/start/cmd.go

```diff
@@ -130,6 +130,28 @@ func enqueueMissingBlocks(exportQueue types.HeightQueue, ctx *parser.Context) {
 		panic(fmt.Errorf("failed to get last block from RPCConfig client: %s", err))
 	}
 
+	lastBlockHeightInDB, err := ctx.Database.GetLastBlockHeight()
+	if err != nil {
+		fmt.Errorf("failed to get last block height from database: %s", err)
```

Comment:
for the error to be printed out: 
```suggestion
		ctx.Logger.Error("failed to get last block height from database", "error", err)
```